src/transport/raw/PeerAddress.h

```diff
@@ -209,14 +267,15 @@ class PeerAddress
         // * 0x00
         // * The 16-bits Group Identifier in big-endian order
         uint32_t groupId = static_cast<uint32_t>((fabric << 24) & 0xff000000) | group;
+
         return UDP(Inet::IPAddress::MakeIPv6PrefixMulticast(scope, prefixLength, prefix, groupId));
     }
 
 private:
-    Inet::IPAddress mIPAddress   = {};
-    Type mTransportType          = Type::kUndefined;
-    uint16_t mPort               = CHIP_PORT; ///< Relevant for UDP data sending.
-    Inet::InterfaceId mInterface = Inet::InterfaceId::Null();
+    IPDestination mDestinations[kMaxPeerDestinations] = {};
+    unsigned mNumValidDestinations                    = 0;
+    Type mTransportType                               = Type::kUndefined;
+    uint16_t mPort                                    = CHIP_PORT; ///< Relevant for UDP data sending.
```

Comment:
Need a TODO to not rely on CHIP_PORT get it from explicit setter and that it has to match the SRV record port

Reply:
This is existing code. I am looking for smaller deltas and don't want to go into "you touched it now you have to fix the whole thing". I am fixing things, but piecewise.